harness/determined/pytorch/_lr_scheduler.py

```diff
@@ -75,11 +75,11 @@ def __init__(
     def step(self, *args: Any, **kwargs: Any) -> None:
         self._scheduler.step(*args, **kwargs)
 
-    def get_last_lr(self) -> List:
-        return self._scheduler.get_last_lr()
+    def get_last_lr(self) -> List[float]:
+        return list(self._scheduler.get_last_lr())
 
     def load_state_dict(self, state_dict: Dict[Any, Any]) -> None:
         self._scheduler.load_state_dict(state_dict)
 
     def state_dict(self) -> Dict[Any, Any]:
-        return self._scheduler.state_dict()
+        return dict(self._scheduler.state_dict())
```

Comment:
Same as last comment, probably better to `type: ignore`. `state_dict()` already returns a dict, PyTorch just doesn't like adding type annotations.

Reply:
In order of preference, I suggest:
1. adding annotations
2. disabling [the specific type error](https://mypy.readthedocs.io/en/stable/error_codes.html)
3. disabling all type errors